score/horizontalpodautoscaler/README.md

```diff
@@ -0,0 +1,3 @@
+Define how to use an `horizontal-pod-autoscaler` resource type for a Workload.
+
+There should be a resource of type `horizontal-pod-autoscaler` defined in your resource definitions, [see examples here](https://developer.humanitec.com/examples/resource-definitions/hpa-driver/horizontalpodautoscaler/).
```

Comment:
I suggest to link to the Template Driver Definition because that one honors the resource inputs from Score, while the hpa Driver based Definition does not.
```suggestion
There should be a resource of type `horizontal-pod-autoscaler` defined in your Resource Definitions which honors the resource inputs from Score, [see examples here](https://developer.humanitec.com/examples/resource-definitions/template-driver/horizontalpodautoscaler/).
```

Reply:
Actually the `hpa` Driver does take into account the `params`. And I think the idea is to show the managed `hpa` Driver and then from there, talk about the custom `template` option. To show that we have pre-built/ready-to-use Drivers?

Reply:
Suggesting to keep it like this.